grpc/runtime/src/main/scala/io/buoyant/grpc/runtime/GrpcStatus.scala

```diff
@@ -82,10 +82,13 @@ object GrpcStatus {
     }
   }
 
-  private def toTrailers(status: GrpcStatus): h2.Frame.Trailers =
-    h2.Frame.Trailers(
-      "grpc-status" -> status.code.toString,
-      "grpc-message" -> status.message
-    )
-
+  private def toTrailers(status: GrpcStatus): h2.Frame.Trailers = {
+    var pairs = Seq("grpc-status" -> status.code.toString)
```

Comment:
TIOLI:
```
if (status.message != null && status.message.nonEmpty) {
  Seq(... both items ...)
} else {
  Seq(... status only...)
}
```
Avoids smelly `var` and might be fewer allocations at the expense of a small amount of code duplication.